http-server/README.md

```diff
@@ -1,21 +1,21 @@
-[![crates.io](https://img.shields.io/crates/v/wascc-httpsrv.svg)](https://crates.io/crates/wascc-httpsrv)&nbsp;
-![Rust](https://github.com/wascc/http-server-provider/workflows/Rust/badge.svg)
-![license](https://img.shields.io/crates/l/wascc-httpsrv.svg)&nbsp;
-[![documentation](https://docs.rs/wascc-httpsrv/badge.svg)](https://docs.rs/wascc-httpsrv)
+[![crates.io](https://img.shields.io/crates/v/wasmcloud-httpserver.svg)](https://crates.io/crates/wasmcloud-httpserver)&nbsp;
+![Rust](https://github.com/wasmcloud/capability-providers/workflows/HTTPSERVER/badge.svg)
```

Comment:
Is this supposed to be all caps? I don't know if github URLs are case sensitive...

Reply:
Funny enough, the workflow URLS _are_ case sensitive, so they have to be all caps (which is what our workflows are)

Reply:
Hah. Did not expect that.